test/GRAPHFILES

```diff
@@ -290,3 +290,5 @@ users/npadmana/twopt/twopt-paircount.graph
 studies/elegance/elegance.graph
 # suite: Samples (bogus)
 Samples/Performance/foo.graph
+#suite: Pimality test
```

Comment:
Typo - this should say "Primality".  Would you move it to before the Samples suite, while you're at it?  If I have any other changes to it, I'll take them on myself (as they will likely be persnickety) :)